### PR TITLE
Update client.js to add RPC suffixes

### PR DESCRIPTION
**Pull request by kimveasna**
### Description
Additional RPC suffixes are appended to manage SOAP RFC from SAP auto-generated webservices.

#### Related issues

- connect to #273 

### Checklist

- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Comment by emonddr** [comment]
@kimveasna , thank you for the fix.

File: src/client.js [hunk 1]
```diff
@@ -333,7 +333,7 @@ class Client extends Base {
           result = obj.Body[outputName];
         }
         if (!result) {
```
Comment (deepakrkris) [inline src/client.js]: some how there are two if blocks implementing the same condition

https://github.com/strongloop/strong-soap/blob/92239c41e2135eeab3c3ae0ec2e4f46253558945/src/client.js#L330-L334

https://github.com/strongloop/strong-soap/blob/92239c41e2135eeab3c3ae0ec2e4f46253558945/src/client.js#L335-L341

along with the changes in this PR we could fix the first block with the regex as well 
`(?:(Out|\.Out)(?:put)?|(Response|\.Response))$`

File: src/client.js [hunk 2]
```diff
@@ -333,7 +333,7 @@ class Client extends Base {
           result = obj.Body[outputName];
```
Comment (deepakrkris) [inline src/client.js]: we can fix the previous regex block as well :
(?:(Out|\.Out)(?:put)?|(Response|\.Response))$

**Comment by emonddr** [comment]
@kimveasna , please squash your commits.
https://loopback.io/doc/en/lb4/submitting_a_pr.html#10-final-rebase-and-squashing-of-commits

e.g.
```
pick ...  
fixup ...
```

**Comment by kimveasna** [comment]
> @kimveasna , please squash your commits.
> https://loopback.io/doc/en/lb4/submitting_a_pr.html#10-final-rebase-and-squashing-of-commits
> 
> e.g.
> 
> ```
> pick ...  
> fixup ...
> ```

Thank you for the instructions. All done.

**Pull request merged**